oamapps/postConfigure/postConfigure.cpp

```diff
@@ -1421,13 +1421,16 @@ int main(int argc, char* argv[])
             mysqlRep = true;
 
         string answer = "y";
+        // MCOL-3888: default 'y', if its something other than 'n' leave it 'y'
+        if ( MySQLRep == "n" )
+            answer = "n";
```

Comment:
Any need to check for 'N' as well?

Reply:
This values has only accepted lowercase 'y' or 'n' until this point and changing this would require changes else where in code that only validates against lowercase values. Seems like a separate ticket would be needed